src/data/security-quiz.ts

```diff
@@ -1,4 +1,38 @@
 const securityQuiz = [
+  {
+    Question:
+      " Which of the following term involves sending packets of data to the target and interpreting their response.?",
+    Answer: "Scanning",
+    Distractor1: "Exploitation",
+    Distractor2: "Scanning",
+    Distractor3: "Reporting",
+    Explanation:
+      "This is the part where you come in contact with the target. Scanning involves sending packets of data to the target and interpreting their response.",
+    Link: "https://www.freecodecamp.org/news/ethical-hacking-lifecycle-five-stages-of-a-penetration-test/",
+
+
+  },
+  {
+    Question:
+      " Getting as much information as you can about the target without interacting directly with the target is known. ?",
+    Answer: "Reconnaissance",
+    Distractor1: "Enumeration",
+    Distractor2: " Exploitation",
+    Distractor3: "Reconnaissance",
```

Comment:
distractors are supposed to be for the wrong answers.
Please change this to something else